docs/proposals/fluentd-secure-forward.md

```diff
@@ -0,0 +1,35 @@
+# Allow Fluentd to secure forward to other Fluentd/Logstash (over ES-Copy)
```

Comment:
I don't think we would change anything with how fluentd is installed; and I'm not sure if we'd want to have a mode that would just install fluentd for scenarios like this... but it sounds like that's what you're thinking here @jcantrill ?

How would we do a normal upgrade for that mode?